### ISSUE TITLE
Project hiatus

### ISSUE THREAD

**Issue by teddy-error** [ruby-mutiny/mutiny#2]
Hi Louis! 

It would seem based on the commit history and domain expiration that the `Mutiny` testing framework became an inactive project as of one year ago. I would like to request, if you don't intend to actively develop this framework that the URL be relinquished for use.

I am working on an organization named `Mutiny` as well which attempts to provide students attending coding bootcamps with open-sourced supplementary material as well as a place to come together in general. We are currently using `team-mutiny` as our URL but we'd prefer to avoid any potential confusion and simply redirect students to `mutiny`. 

TL;DR: we'd love to use your URL, if she'll have us back :)

**Comment by louismrose** [comment]
Hi @teddy-error,

That's right -- I'm not actively developing Mutiny at the moment, so more than happy to pass on the organisation name. I'll rename the organisation that owns this project shortly, and then mutiny should be free for you to pick up.

**Comment by teddy-error** [comment]
Thank you for your kindness, @louismrose
